reference/opt/shaders-msl/vert/basic.capture.vert

```diff
@@ -0,0 +1,35 @@
+#include <metal_stdlib>
+#include <simd/simd.h>
+
+using namespace metal;
+
+struct UBO
+{
+    float4x4 uMVP;
+};
+
+struct spvAux
+{
+    uint vertexCount;
+    uint swizzleConst[1];
+};
+
+struct main0_out
+{
+    float3 vNormal [[user(locn0)]];
```

Comment:
Should outputs have locations/semantics when they are captured?

Reply:
Do we have any way of dealing with XfbStride/XfbOffset for VK_EXT_transform_feedback?

Reply:
> Should outputs have locations/semantics when they are captured?

Probably not, since they aren't used.

> Do we have any way of dealing with XfbStride/XfbOffset for VK_EXT_transform_feedback?

Not yet. That's definitely something we'll need to add, though.

Reply:
Ok, don't think it's necessary now at any rate.